extension_helpers/_openmp_helpers.py

```diff
@@ -224,9 +224,10 @@ def is_openmp_supported():
     """
     Determine whether the build compiler has OpenMP support.
     """
-    log_threshold = log.set_threshold(log.FATAL)
+    log_threshold = log.level
+    log.setLevel('CRITICAL')
```

Comment:
Not 100% sure if this is the equivalent - setuptools doesn't have a common log like distutils does